packages/platforms/delta/x86-64/x86-64-delta-agc7648a/modules/builds/tmp421.c

```diff
@@ -0,0 +1,309 @@
+/* tmp421.c
```

Comment:
@hans-tseng  The TMP421 driver is already available in the kernel. It should just be enabled in the kernel configuration. If there any reason you've placed it here as a separate platform module?